src/data/image.cpp

```diff
@@ -73,7 +73,7 @@ void Image::insertImage(
     throw invalid_argument("Source image doesn't fit");
   }
 
-  auto sourceIter = pixels.cbegin();
+  auto sourceIter = pixels.begin();
```

Comment:
any particular reason of this change even when you are not modifying iter ?

Reply:
Yes: If you have a `const vector` or other container, then `begin` and `end` will be equivalent to `cbegin`/`cend`. If you look at the [reference](https://en.cppreference.com/w/cpp/container/vector/begin) for `std::vector::begin`, for example, you can see that there are two overloads for `begin`:

```cpp
iterator begin() noexcept;
const_iterator begin() const noexcept;
const_iterator cbegin() const noexcept;
```

If `begin` is called on a `const` `vector`, the 2nd overload is chosen, and a `const_iterator` is returned. 

The case where `cbegin`/`cend` are useful is if you have a _non_ `const` container, but you want to get a `const_iterator` to it. When I originally wrote this code a couple years ago, this wasn't fully clear to me, so I used `cbegin`/`cend` everywhere. But that's not necessary, so I thought I would clean it up.